19.03.5.sh

```diff
@@ -511,6 +518,32 @@ do_install() {
 			)
 			exit 0
 			;;
+		sles)
+			(
+			set -x
+			$sh_c 'zypper refresh'
+			docker_pkg=$(zypper se -n -s docker | grep -E "\sdocker\s.*${docker_version//\./\\.}.*$architecture" | sort --sort=v -k3 | tail -1 | cut -d '|' -f4 | cut -d ' ' -f2)
+			if [ -z "$docker_pkg" ]; then
+			    trunc_docker_version=${docker_version%.*}
+				available_docker_pkgs=$(zypper se -n -s docker | grep -E "\sdocker\s.*${trunc_docker_version//\./\\.}.*$architecture" | sort --sort=v -k3 | cut -d '|' -f4 | cut -d ' ' -f2 | cut -d '_' -f1)
+				cat >&2 <<-EOF
+				No match found for SLES that matches docker $docker_version for $architecture.
```

Comment:
Can we check if anything was found, the output on systems that have no Docker available at all:

```
No match found for SLES that matches docker 19.03.5 for x86_64.
                                                                                                                                                                                                                                                                                                                         
SLES 12.5 only supports the following patches within docker 19.03: 

```

Can we also use `Docker` as its a product name

Reply:
Added this to the latest commit. On failure, it would output:
`SLES 12.5 does not support any patches for docker 19.03 for x86_64.`